app/ide-desktop/lib/dashboard/src/layouts/Settings.tsx

```diff
@@ -110,16 +114,22 @@ export default function Settings(props: SettingsProps) {
             />
           </aria.Popover>
         </aria.MenuTrigger>
-        <aria.Text className="py-heading-y">{getText('settingsFor')}</aria.Text>
+        <ariaComponents.Text variant="h1" className="py-heading-y">
+          {getText('settingsFor')}
+        </ariaComponents.Text>
         {/* This UI element does not appear anywhere else. */}
         {/* eslint-disable-next-line no-restricted-syntax */}
-        <div className="ml-[0.625rem] h-[2.25rem] rounded-full bg-frame px-[0.5625rem] pb-[0.3125rem] pt-[0.125rem] leading-snug">
+        <ariaComponents.Text
+          variant="h1"
+          truncate="1"
+          className="ml-[0.625rem] h-[2.25rem] max-w-[30ch] rounded-full bg-frame px-[0.5625rem] pb-[0.3125rem] pt-[0.125rem] leading-snug"
```

Comment:
Btw, would like to take a look at this change? https://github.com/enso-org/enso/pull/10171/files#diff-b4033a983460df383c4932fac422821a9066585230aedee437e2540f2e4da514R146 

I did it for the Billing page(it also fixes nasty overflowing bugs in settings). Maybe we can cherry pick these changes instead?